### PR TITLE
fix: nft waterfalls requests

### PR DESCRIPTION
**Pull request by zzmp**
- Re-architects NFT queries to load them from a parent, to avoid lazy-loading causing waterfalled requests when there are multiple requests on a single screen.

There are a few strategies to do this. In this case, I only used a parent component to trigger the load, and then read that data using `store-only` from the component using it.

_Normally_, you'd want the screen to trigger _all_ loads for a page (although lazy-loading sweep is an exception), and you'd want to pass the queryRef down the component tree. However, passing the `queryRef` is cumbersome and confusing, so I instead elected to lazy-load (to avoid re-architecting `<Suspense />` usage) and just read from `store-only` (to avoid triggering redundant requests).

### CODE REVIEW

File: src/nft/components/explore/CarouselCard.tsx [hunk 1]
```diff
@@ -191,6 +191,7 @@ const MARKETS_ENUM_TO_NAME = {
 }
 
 export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
+  useLoadCollectionQuery(collection.address)
```
[the remark this answers is not in the document]
Comment (zzmp) [reply src/nft/components/explore/CarouselCard.tsx:194]: Do we _need_ to preload these 5 trending collections? I don't think we should load anything without an explicit user action, unless we know they'll end up here.

[the remark this answers is not in the document]
Comment (zzmp) [reply src/nft/components/explore/CarouselCard.tsx]: Updated this to preload through the parent Banner in 5c8ee29

File: src/graphql/data/nft/Asset.ts [hunk 2]
```diff
@@ -178,22 +178,42 @@ function formatAssetQueryData(queryAsset: NftAssetsQueryAsset, totalCount?: numb
...
-    [address, after, asc, before, filter, first, last, orderBy]
-  )
-  const [queryOptions, setQueryOptions] = useState({ fetchKey: 0 })
-  const queryData = useLazyLoadQuery<AssetQuery>(assetQuery, vars, queryOptions)
+}
+
+const defaultAssetFetcherParams: Omit<AssetQuery$variables, 'address'> = {
+  orderBy: 'PRICE',
+  asc: true,
+  // tokenSearchQuery must be specified so that this exactly matches the initial query.
```
Comment (cbachmeier) [inline src/graphql/data/nft/Asset.ts:197]: good catch

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**